### PR TITLE
Update dependent operator versions in kube_setup.sh

### PR DESCRIPTION
**Pull request by kylape**
Required to support drop of v1beta1 CRD API

### CODE REVIEW

File: build/kube_setup.sh [hunk 1]
```diff
@@ -63,7 +63,7 @@ mkdir -p "$DOWNLOAD_DIR"
 
 
 function install_strimzi_operator {
-    STRIMZI_VERSION=0.22.1
+    STRIMZI_VERSION=0.25.0
```
Comment (bsquizz) [inline build/kube_setup.sh:66]: We went to v0.24.0 as of here: https://github.com/RedHatInsights/clowder/commit/a6d7f44c5218f2a6877e5cb151e2cbd4a38eda7c#diff-3f7aa29c9a9e65ce82ef5763d1a798a1aaf42b5edb119f4e7df1c9d44b0840bfR66

File: build/kube_setup.sh [hunk 2]
```diff
@@ -252,7 +252,8 @@ function install_elasticsearch_operator {
     fi
 
     echo "*** Applying elastic-operator manifest ..."
-    ${KUBECTL_CMD} apply -f https://download.elastic.co/downloads/eck/1.6.0/all-in-one.yaml
+    ${KUBECTL_CMD} create -f https://download.elastic.co/downloads/eck/1.7.1/crds.yaml
```
Comment (bsquizz) [inline build/kube_setup.sh:255]: Any reason for using `create` vs `apply`?

**Comment by psav** [comment]
Needs fix after strimzi was updated

File: build/kube_setup.sh (hunk 2 above)
Comment (kylape) [reply build/kube_setup.sh:255]: Pulled instructions from https://www.elastic.co/downloads/elastic-cloud-kubernetes

**Pull request merged**